content/exchange.html

```diff
@@ -61,81 +37,17 @@ <h2>SE Abroad</h2>
     <div id="exchange-countries-map"></div>
     <div id="exchange-map-text" class="center"></div>
 </div>
-<!-- TODO: since the countries are so small, have a list on the side indicating which
-     schools that can also zoom the map -->
 <div class="vbx2"></div>
 
-<h3> The Exchange Experience </h3>
+<h2> The Exchange Experience </h2>
 <p class="full">
-    <span class="big">
-        "An invaluable opportunity to travel and to immerse yourself in a different
-        culture."
-    </span>
     We asked our exchangees to provide us some insight into their exchange experience.
-    <span class="small">
-        Many of the sentences are directly quoted or rephrased from the original submissions.
-    </span>
-</p>
-<div class="vb"></div>
-
-<h5>What was the hardest thing about going on exchange?</h5>
-<p class="quote">
-    The paperwork and the course planning.
-    <br /><br />
-    The language barrier - the area is mostly French-speaking.
-    <br /><br />
-    Finding a co-op for the next term, especially if the term start/end dates did
-    not match the typical UW start/end dates.
-    <br /><br />
-    Connecting with the locals and making friends in a meaningful capacity. Some
-    exchange schools have a more mature program than others where it is easier to
-    make friends. If I were to do exchange again I would opt for a school where a
-    lot of international students go and where the program is well established.
-    <br /><br />
-    Balancing schoolwork/wanting to graduate at the same time as the SE cohort with
-    taking advantage of being in a new country. Oh, and the timezone differences when
-    communicating with friends, family, group project members and North American
-    interviewers was a pain.
 </p>
 <div class="vb"></div>
-
-<h5>What about something that was easier to do for/while on exchange?</h5>
-<p class="quote">
-    Applying for exchange.
-    <br /><br />
-    Less time spent on school, more time available for travelling.
-    <br /><br />
-    Having fun was easier.
-    <br /><br />
-    Studies at my exchange program were significantly easier than at UWaterloo.
-</p>
+<h3>What was your favourite part of exchange?</h3>
```

Comment:
The font for this graph title, and the next one looks very different. Could we update that styling to match other graph titles?

Reply:
good catch!